.travis.yml

```diff
@@ -4,16 +4,17 @@ sudo: false
 env:
   global:
     - ELIXIR_ASSERT_TIMEOUT=2000
+    - LATEST_OTP_RELEASE=false
```

Comment:
I would suggest not setting this default value here. And then you can do this in the script:

    - if [ -n CHECK_REPRODUCIBLE ]; then make check_reproducible; fi

See that I am also proposing to rename the variable, as I think that better reveals the intent. Thoughts?

Reply:
sure. I think I chose this one when we were going to check if it was reproducible and also the future test..